packages/sdks/src/blocks/img/component-info.ts

```diff
@@ -2,21 +2,154 @@ import { IMAGE_FILE_TYPES } from '../../constants/file-types.js';
+
+            img.src = url;
+            setTimeout(() => {
+              if (!loaded) {
+                reject(new Error('Image load timed out'));
+              }
+            }, timeout);
+          });
+        }
+
+        function round(num: number) {
+          return Math.round(num * 1000) / 1000;
+        }
+
+        const value = options.get('image');
+        const aspectRatio = options.get('aspectRatio');
+
+        // For SVG images - don't render as webp, keep them as SVG
+        fetch(value)
+          .then((res) => res.blob())
+          .then((blob) => {
+            if (blob.type.includes('svg')) {
+              options.set('noWebp', true);
+            }
+          });
+
+        if (value && (!aspectRatio || aspectRatio === DEFAULT_ASPECT_RATIO)) {
+          return loadImage(value).then((img) => {
+            const possiblyUpdatedAspectRatio = options.get('aspectRatio');
+            if (
+              options.get('image') === value &&
+              (!possiblyUpdatedAspectRatio ||
+                possiblyUpdatedAspectRatio === DEFAULT_ASPECT_RATIO)
+            ) {
+              if (img.width && img.height) {
+                options.set('aspectRatio', round(img.height / img.width));
+                options.set('height', img.height);
+                options.set('width', img.width);
+              }
+            }
+          });
+        }
+      },
+    },
+    {
+      name: 'backgroundSize',
+      type: 'text',
+      defaultValue: 'cover',
+      enum: [
+        {
+          label: 'contain',
+          value: 'contain',
+          helperText: 'The image should never get cropped',
+        },
+        {
+          label: 'cover',
+          value: 'cover',
+          helperText: "The image should fill it's box, cropping when needed",
+        },
+      ],
+    },
+    {
+      name: 'backgroundPosition',
+      type: 'text',
+      defaultValue: 'center',
+      enum: [
+        'center',
+        'top',
+        'left',
+        'right',
+        'bottom',
+        'top left',
+        'top right',
+        'bottom left',
+        'bottom right',
+      ],
+    },
+    {
+      name: 'altText',
+      type: 'string',
+      helperText: 'Text to display when the user has images off',
+    },
+    {
+      name: 'title',
+      type: 'string',
+      helperText: 'Text to display when hovering over the asset',
+    },
+    {
+      name: 'aspectRatio',
+      type: 'number',
+      helperText:
+        "This is the ratio of height/width, e.g. set to 1.5 for a 300px wide and 200px tall photo. Set to 0 to not force the image to maintain it's aspect ratio",
+      advanced: true,
+      defaultValue: 0.7041,
+    },
+    {
+      name: 'fitContent',
+      type: 'boolean',
+      helperText:
+        "When child blocks are provided, fit to them instead of using the image's aspect ratio",
+      defaultValue: true,
     },
   ],
+
+  shouldReceiveBuilderProps: {
+    builderBlock: true,
+  },
   noWrap: true,
```

Comment:
is the 'Raw:Img' exactly same as the 'Image' now?

Reply:
should we re-export the Image itself for Raw:Img then? instead of managing them both? tagging @samijaber here